pkg/controllers/cloud/account_poller.go

```diff
@@ -141,26 +144,33 @@ func (p *Poller) addAccountPoller(cloudType cloudv1alpha1.CloudProvider, namespa
 	return poller, false
 }
 
-// removeAccountPoller removes an account poller from accPollers map.
+// removeAccountPoller stops go-routine and removes an account poller from accPollers map.
```

Comment:
mention what thread is getting stopped.. avoid goroutine

Reply:
Done